backend/sftp/sftp.go

```diff
@@ -101,16 +102,16 @@ func Create(dir string, program string, args ...string) (*SFTP, error) {
 
 	dirs := []string{
 		dir,
-		filepath.Join(dir, backend.Paths.Data),
-		filepath.Join(dir, backend.Paths.Snapshots),
-		filepath.Join(dir, backend.Paths.Index),
-		filepath.Join(dir, backend.Paths.Locks),
-		filepath.Join(dir, backend.Paths.Keys),
-		filepath.Join(dir, backend.Paths.Temp),
+		Join(dir, backend.Paths.Data),
+		Join(dir, backend.Paths.Snapshots),
+		Join(dir, backend.Paths.Index),
+		Join(dir, backend.Paths.Locks),
+		Join(dir, backend.Paths.Keys),
+		Join(dir, backend.Paths.Temp),
 	}
 
 	// test if config file already exists
-	_, err = sftp.c.Lstat(filepath.Join(dir, backend.Paths.Config))
+	_, err = sftp.c.Lstat(dir + backend.Paths.Config)
```

Comment:
Why did you remove the `path.Join()` here? is `dir` guaranteed to always end with a slash?